### ISSUE TITLE
Another place to store FTP login details?

### ISSUE THREAD

**Issue by bighairydave** [dmathieu/glynn#35]
Is it possible to store FTP login details somewhere other than config.yml? (reason: for my new site I want my config.yml to go in a public git repository, but I still want to use Glynn).

One solution would be to be able to tell Glynn to read its settings from another file that can be excluded from the repository.

If I have overlooked a way to get Jekyll to read in options from additional files after parsing  config.yml, that would be a solution too.

**Comment by dmathieu** [comment]
We directly get jekyll's configuration object.
https://github.com/dmathieu/glynn/blob/master/lib/glynn/jekyll.rb#L8

So, if jekyll handles, we handle it. The location of that file isn't within the scope of this repository.

**Comment by bcole808** [comment]
I was also in need of this feature because I want my primary config.yml to be published in a public repository, but I need the deployment credentials to be kept secret. I searched around for a way in Jekyll to put some configuration options in a separate file, but could not find a good solution. 

I added a feature to optionally look for additional configuration options in a file called `_glynn.yml` in order to achieve this. I will open a pull request shortly in case this feature can be merged back into the project for others. 

With this implementation, the presence of a `_glynn.yml` file is optional and the FTP configuration items can be placed either in `_config.yml` or in `_glynn.yml` and the latter will be preferred if present.
